main.go

```diff
@@ -152,12 +153,23 @@ func main() {
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
 	if renderMode {
-		if err := renderObjects(
-			renderManifests(
-				rteManifests,
-				renderNamespace,
-				renderImage,
-			).ToObjects()); err != nil {
+		var objs []client.Object
+		if enableScheduler {
```

Comment:
We'll only render the scheduler manifests here without the RTE one.
is that what we want?

Reply:
Actually no because this is a different semantic than without the `-render` flag enabled. Let me fix it.